sal_interfaces/SALGenerics.xml

```diff
@@ -162,23 +170,24 @@
     </SALEvent>
     <SALEvent>
       <Subsystem>SALGeneric</Subsystem>
-      <EFDB_Topic>SALGeneric_logevent_summaryState</EFDB_Topic>
```

Comment:
i'm not sure why it is saying I changed this. Some cut/paste issues I think. You'll see it is in line 185 in the new version.